src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -75,49 +78,65 @@ export const RegistrationFormActions = ({
         }}>
         {tabNumber > RegistrationTab.Description && (
```

Comment:
Siden vi nå har en variabel som i praksis er det samme
```suggestion
        {!isFirstTab && (
```

Reply:
Ser nå at bruken av `isFirstTab` ikke trengs under her heller, så da kan vi forsåvidt fjerne de 2 nye constene, og ta det slik det var også. Går for det samme